sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -65,20 +65,10 @@ export class UnavailableDefaultCredential implements TokenCredential {
  * - {@link AzureCliCredential}
  * - {@link AzurePowerShellCredential}
  * - {@link AzureDeveloperCliCredential}
+ * - {@link BrokerCredential} - A broker-enabled instance of {@link InteractiveBrowserCredential}
```

Comment:
In JS, BrokerCredntial is not an instance of InteractiveBrowserCredential, so I don’t think we need to link them together.
```suggestion
 * - {@link BrokerCredential}
```